kolibri/plugins/user_profile/assets/src/routes.js

```diff
@@ -52,68 +56,74 @@ export default [
     children: [
       {
         path: 'change',
-        name: 'CHANGE_FACILITY',
-        component: ChangeFacility,
+        name: 'SELECT_FACILITY',
+        component: SelectFacility,
       },
+      {
+        path: 'change',
```

Comment:
Was using the same `path: 'change'` twice, for both `SELECT_FACILITY` and `CONFIRM_CHANGE_FACILITY`, intentional?